Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py

```diff
@@ -350,7 +351,41 @@ def testSpiroAndBridgeheads(self):
     self.assertEqual(len(sa),2)
     self.assertEqual(sorted(sa),[6,9])
     
-
-
+  def testNumRotatableBonds(self):
+    self.assertEquals(-1,
+           int(rdMD.NumRotatableBondsOptions.Default))
+    self.assertEquals(0,
+           int(rdMD.NumRotatableBondsOptions.NonStrict))
+    self.assertEquals(1,
+           int(rdMD.NumRotatableBondsOptions.Strict))
+    self.assertEquals(2,
+           int(rdMD.NumRotatableBondsOptions.StrictLinkages))
+    
+    for s in [#"C1CC1CC",
+              "CCNC(=O)NCC",
+              #'Cc1cccc(C)c1c1c(C)cccc1C',
+              #'CCc1cccc(C)c1c1c(C)cccc1CC',
+              #'Cc1cccc(C)c1c1c(C)nccc1C',
+              'Cc1cccc(C)c1c1c(C)cccc1',
+              #'CCO',
+    ]:
+              
+      m = Chem.MolFromSmiles(s)
+
+      v1 = rdMD.CalcNumRotatableBonds(m)
+      
+      v2 = rdMD.CalcNumRotatableBonds(m, False)
+      v3 = rdMD.CalcNumRotatableBonds(m, True)
+
+      v4 = rdMD.CalcNumRotatableBonds(m, rdMD.NumRotatableBondsOptions.Default)
+      v5 = rdMD.CalcNumRotatableBonds(m, rdMD.NumRotatableBondsOptions.NonStrict)
+      v6 = rdMD.CalcNumRotatableBonds(m, rdMD.NumRotatableBondsOptions.Strict)
+      v7 = rdMD.CalcNumRotatableBonds(m, rdMD.NumRotatableBondsOptions.StrictLinkages)
+      
+      self.assertEquals(v1, v4)
+      self.assertEquals(v2, v5)
+      self.assertEquals(v3, v6)
+      print (v1, v2, v3, v4, v5, v6, v7)
```

Comment:
The print statement is, the rest are a regression test to assure that the True/False does the correct thing when compared to the enum.  I'm assuming you were referring to the print statement.